ndb/tests/unit/test_eventloop.py

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import collections
 import pytest
+import unittest.mock
```

Comment:
Can you separate std. library imports from third-party?

```python
import collections
import unittest.mock

import pytest
```

Reply:
95a94f4bd0753a77890a3b07648b13845c296ac5